mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -29,47 +39,59 @@ def sst_timeseries(config):
     # reading only those that are between the start and end dates
     startDate = config.get('time', 'timeseries_start_date')
     endDate = config.get('time', 'timeseries_end_date')
-    infiles = streams.readpath('timeSeriesStatsOutput',
-                               startDate=startDate, endDate=endDate)
-    print 'Reading files {} through {}'.format(infiles[0],infiles[-1])
-
+    streamName = streams.find_stream(streamMap['timeSeriesStats'])
+    infiles = streams.readpath(streamName, startDate=startDate,
+                               endDate=endDate)
+    print 'Reading files {} through {}'.format(infiles[0], infiles[-1])
 
-    casename = config.get('case','casename')
-    ref_casename_v0 = config.get('case','ref_casename_v0')
-    indir_v0data = config.get('paths','ref_archive_v0_ocndir')
+    casename = config.get('case', 'casename')
+    ref_casename_v0 = config.get('case', 'ref_casename_v0')
+    indir_v0data = config.get('paths', 'ref_archive_v0_ocndir')
 
-    plots_dir = config.get('paths','plots_dir')
+    plots_dir = config.get('paths', 'plots_dir')
 
-    yr_offset = config.getint('time','yr_offset')
+    yr_offset = config.getint('time', 'yr_offset')
 
-    N_movavg = config.getint('sst_timeseries','N_movavg')
+    N_movavg = config.getint('sst_timeseries', 'N_movavg')
 
     regions = config.getExpression('regions', 'regions')
     plot_titles = config.getExpression('regions', 'plot_titles')
     iregions = config.getExpression('sst_timeseries', 'regionIndicesToPlot')
 
     # Load data:
-    ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
-                           preprocess_mpas(x, yearoffset=yr_offset,
-                                           timestr=['xtime_start', 'xtime_end'],
-                                           onlyvars=['time_avg_avgValueWithinOceanRegion_avgSurfaceTemperature']))
+    varList = ['avgSurfaceTemperature']
```

Comment:
@pwolfram, if the variable name changed on the MPAS dycore side, we would need to add the new name to the appropriate dictionary.  If we decided to change the name here from `avgSurfaceTemperature` to `avgSeaSurfaceTemperature`, the corresponding key in the `variableMap` would need to be renamed but that would be it.  

By the way, there's no reason that the variable name here has to follow the name in the MPAS dycore, though it's presumably helpful.